app/lib/theme/CustomDiffFields.tsx

```diff
@@ -20,193 +23,161 @@ const contentSuffixElement = (
-    : "diffReviewAndSubmitInformationNew";
 
-  return (
-    <>
-      <span id={id && `${id}-${diffClsName}`} className={diffClsName}>
-        {isDate ? getLocaleFormattedDate(newData) : newData}
-      </span>
-      {contentSuffix && contentSuffixElement(id, contentSuffix)}
-      {isAmendmentsAndOtherRevisionsSpecific ? (
-        <span>
-          <em>(ADDED)</em>
-          <style jsx>{`
-            span {
-              color: #cd2026;
-            }
-          `}</style>
+  // Case 2 ->  010
+  if (!oldData && newData && !latestCommittedData) {
+    return (
+      <>
+        <span id={id && `${id}-${diffTextClsName}`} className={diffTextClsName}>
+          {"Not Entered"}
         </span>
-      ) : (
-        <>
-          <FontAwesomeIcon
-            className={"diff-arrow"}
-            size="lg"
-            color="black"
-            icon={faLongArrowAltRight}
-          />
-          <span>
-            <strong>
-              <em>ADDED</em>
-            </strong>
-          </span>
-        </>
-      )}
+        {renderArrow()}
+        <span id={id && `${id}-${diffNewClsName}`} className={diffNewClsName}>
+          {renderData(isDate, newData)}
+        </span>
+        {contentSuffix && contentSuffixElement(id, contentSuffix)}
+      </>
+    );
+  }
 
-      {
-        //   latestCommittedData &&
-        overwriteNotification({ latestCommittedData, isDate })
-      }
-    </>
-  );
-};
+  // Case 1 ->  001
+  if (!oldData && !newData && latestCommittedData) {
+    return <>ERROR SD: 1</>;
+  }
 
-const showStringRemoved = (
-  id: string,
-  oldData: string,
-  latestCommittedData: string,
-  isDate: boolean = false,
-  contentSuffix?: string,
-  isAmendmentsAndOtherRevisionsSpecific?: boolean
-): JSX.Element => {
-  const diffClsName = isAmendmentsAndOtherRevisionsSpecific
-    ? "diffAmendmentsAndOtherRevisionsOld"
-    : "diffReviewAndSubmitInformationOld";
-  return (
-    <>
-      <span id={id && `${id}-${diffClsName}`} className={diffClsName}>
-        {isDate ? getLocaleFormattedDate(oldData) : oldData}
-      </span>
-      {contentSuffix && contentSuffixElement(id, contentSuffix)}
-      {!isAmendmentsAndOtherRevisionsSpecific && (
-        <>
-          <FontAwesomeIcon
-            className={"diff-arrow"}
-            size="lg"
-            color="black"
-            icon={faLongArrowAltRight}
-          />
-          <span>
-            <strong>
-              <em>REMOVED</em>
-            </strong>
-          </span>
-        </>
-      )}
-      {
-        // latestCommittedData && oldData && oldData !== latestCommittedData &&
-        overwriteNotification({ latestCommittedData, isDate })
-      }
-    </>
-  );
+  // Case 0 ->  000
+  if (!oldData && !newData && !latestCommittedData) {
+    return (
+      <>
+        <span id={id && `${id}-${diffTextClsName}`} className={diffTextClsName}>
+          {"Not Entered"}
+        </span>
+      </>
+    );
+  }
 };
 
 const showNumberDiff = (
```

Comment:
I'll look at the NumberField in more detail in the next round of review--at this point it's pretty much all the same comments as the StringField's